hazelcast/src/main/java/com/hazelcast/internal/management/operation/ScriptExecutorOperation.java

```diff
@@ -79,31 +71,14 @@ public Object getResponse() {
     protected void writeInternal(ObjectDataOutput out) throws IOException {
         out.writeUTF(engineName);
         out.writeUTF(script);
-        if (bindings != null) {
-            out.writeInt(bindings.size());
-            Set<Map.Entry<String, Object>> entries = bindings.entrySet();
-            for (Map.Entry<String, Object> entry : entries) {
-                out.writeUTF(entry.getKey());
-                out.writeObject(entry.getValue());
-            }
-        } else {
-            out.writeInt(0);
-        }
+        // kept for compatibility
+        out.writeInt(0);
     }
 
     @Override
     protected void readInternal(ObjectDataInput in) throws IOException {
         engineName = in.readUTF();
         script = in.readUTF();
```

Comment:
Here we still have to consume the `int` that was written from `writeInternal` even if it's not used any more. Otherwise the next packet will be misaligned and serialization exceptions will occur.